CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.56.0
+
+- Packit can now build RPMs in mock. For more information see https://packit.dev/docs/cli/build/mock (#1662)
+- Packit now provides a more helpful error message when it hits a known issue while creating a Bodhi update: fedora-infra/bodhi#4660 (#1660)
+- Packit now correctly supports `tmt_plan` and `tf_post_install_script` in the configuration. (#1659)
+- RPM build commands of Packit CLI have been merged into one build subcommand, for more information see the updated documentation at https://packit.dev/docs/cli/build/. We have also introduced a new `--srpm` option to the new build subcommand that can be used to trigger local, Copr or Koji build from an already built SRPM rather than implicitly created one by Packit. (#1611)
```

Comment:
I used slightly different wording in the blog post, I think it's better: https://github.com/packit/packit.dev/pull/503/files#diff-31b126523ba267c47d3cfcda884f0a85de7702f6395e9f3e793d6d72905eb900R30-R32

Reply:
will update, thanks